pkg/cmd/cli/snapshotlocation/get.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package snapshotlocation
+
+import (
+	api "github.com/heptio/ark/pkg/apis/ark/v1"
+	"github.com/heptio/ark/pkg/client"
+	"github.com/heptio/ark/pkg/cmd"
+	"github.com/heptio/ark/pkg/cmd/util/output"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewGetCommand(f client.Factory, use string) *cobra.Command {
+	var listOptions metav1.ListOptions
+	c := &cobra.Command{
+		Use:   use,
+		Short: "Get snapshot locations",
+		Run: func(c *cobra.Command, args []string) {
+			err := output.ValidateFlags(c)
+			cmd.CheckError(err)
+			arkClient, err := f.Client()
+			cmd.CheckError(err)
+			var locations *api.VolumeSnapshotLocationList
+			if len(args) > 0 {
+				locations = new(api.VolumeSnapshotLocationList)
+				for _, name := range args {
+					location, err := arkClient.Ark().VolumeSnapshotLocations(f.Namespace()).Get(name, metav1.GetOptions{})
+					cmd.CheckError(err)
+					locations.Items = append(locations.Items, *location)
+				}
+			} else {
+				locations, err = arkClient.ArkV1().VolumeSnapshotLocations(f.Namespace()).List(listOptions)
+				cmd.CheckError(err)
+			}
+			_, err = output.PrintWithFormat(c, locations)
```

Comment:
take a look at this method's implementation - you'll need to add a handler for VolumeSnapshotLocations

Reply:
Added, was there something we wanted to display here other than name and provider? I am sure there is but I couldn't really determine what output we wanted from the structs.